packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.module.scss

```diff
@@ -0,0 +1,119 @@
+@use '@/scss/underscore' as _;
+@use '@/scss/dimensions' as dim;
+
+.currentTenantCard {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: _.unit(1);
+  margin-left: - _.unit(1);
```

Comment:
What is this for?

Reply:
Should've been '_.unit(-1)`

Reply:
The tenant selector card has a 1px padding, as a result, the visual distance between the card content (when not hovered) and the divider bar will be 5px, which is not expected. Old cha told me this lol